client/src/App.tsx

```diff
@@ -117,6 +117,10 @@ const App = () => {
     return localStorage.getItem("lastBearerToken") || "";
   });
 
+  const [headerName, setHeaderName] = useState<string>(() => {
+    return localStorage.getItem("lastHeaderName") || "";
```

Comment:
why is this called `lastHeaderName` ? what does `last` signify here ?

Reply:
I just kept the convention of `lastBearerToken`. I assumed `last` means "last inserted value" (which was stored).